getting-started/securing-timescale-cloud.md

```diff
@@ -0,0 +1,68 @@
+# How to secure network access to your Timescale Cloud service
```

Comment:
Title is a bit long:

<img width="1956" alt="Screen Shot 2020-02-29 at 11 05 30 PM" src="https://user-images.githubusercontent.com/12288273/75619283-10738300-5b48-11ea-9817-b479d7328555.png">